src/dc_msg.rs

```diff
@@ -479,19 +478,25 @@ pub fn dc_msg_load_from_db<'a>(msg: *mut dc_msg_t<'a>, context: &'a Context, id:
                 (*msg).type_0 = row.get(12)?;
                 (*msg).state = row.get(13)?;
                 (*msg).is_dc_message = row.get(14)?;
-                (*msg).text = row.get::<_, String>(15).unwrap_or_default().strdup();
+                (*msg).text = Some(row.get::<_, String>(15)?);
```

Comment:
blocking: this should be `row.get::<_Option<String>>(15)?` instead
otherwise it will panic on null things in the db, and for now it seems text is optional

Reply:
Ok.